Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shuttle_records.yml

```
@@ -0,0 +1,49 @@
+﻿- type: entity
+  name: shuttle records console
+  parent: [BaseStructureDisableToolUse, BaseStructureIndestructible, BaseComputer]
+  id: ComputerShuttleRecordsBase
+  description: Used to recover lost ID cards
```

Marginally more descriptive?
```suggestion
  description: Used to browse shuttle records and recover or reassign shuttle ownership to IDs.
```